apis/swagger.yml

```diff
@@ -1362,6 +1394,93 @@ definitions:
         format: "int64"
         description: "The length of the file dfget requests to download in bytes."
 
+  NetworkInfoFetchRequest:
+    type: "object"
+    description: ""
+    properties:
+      urls:
+        type: "array"
+        description: |
+          The urls is to filter the peer node, the url should be match with taskURL in TaskInfo.
+        items:
+          type: "string"
+
+  Extra:
+    type: "object"
+    description: ""
+    properties:
+      site:
+        type: "string"
+      room:
+        type: "string"
+      rack:
+        type: "string"
+      idc:
+        type: "string"
+
+  TaskFetchInfo:
+    type: "object"
+    description: ""
+    properties:
+      task:
+        $ref: "#/definitions/TaskInfo"
+      pieces:
+        type: "array"
+        description: "The pieces which should belong to the peer node"
+        items:
+          $ref: "#/definitions/PieceInfo"
+
+  Node:
+    type: "object"
+    description: ""
+    properties:
+      basic:
+        $ref: "#/definitions/PeerInfo"
+        description: "Basic node info"
+      extra:
```

Comment:
I think the `extra` property is a map struct not a pre-defined `object`, thus we can add some customized properties without changing the `api` in the future.

Reply:
done.